Update CI/CD (#2091)
kamranahmedse/developer-roadmap: content/roadmaps/113-qa/content/110-qa-ci-cd/readme.md

> # QA CI / CD
> 
> ## What is Continuous Integration (CI)?
> 
> Continuous Integration is a software development method where team members integrate their work at least once a day. In this method, every integration is checked by an automated build to detect errors. In Continuous Integration after a code commit, the software is built and tested immediately. In a large project with many developers, commits are made many times during a day. With each commit code is built and tested.
> 
> ## What is Continuous Delivery (CD)?
> 
> Continuous Delivery is a software engineering method in which a team develops software products in a short cycle. It ensures that software can be easily released at any time. The main aim of continuous delivery is to build, test, and release software with good speed and frequency. It helps you to reduce the cost, time, and risk of delivering changes by allowing for frequent updates in production.
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/continuous-integration.html'>What is CI/CD? Continuous Integration and Continuous Delivery</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/continuous-integration-vs-delivery-vs-deployment.html'>Continuous Integration vs Delivery vs Deployment</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/ci-cd-pipeline.html'>CI/CD Pipeline: Learn with Example</BadgeLink>

Reviewer: ```suggestion
# CI / CD

Continuous Integration is a software development method where team members integrate their work at least once daily. An automated build checks every integration to detect errors in this method. In Continuous Integration, the software is built and tested immediately after a code commit. In a large project with many developers, commits are made many times during the day. With each commit, code is built and tested.

Continuous Delivery is a software engineering method in which a team develops software products in a short cycle. It ensures that software can be easily released at any time. The main aim of continuous delivery is to build, test, and release software with good speed and frequency. It helps reduce the cost, time, and risk of delivering changes by allowing for frequent updates in production.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/continuous-integration.html'>What is CI/CD? Continuous Integration and Continuous Delivery</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/continuous-integration-vs-delivery-vs-deployment.html'>Continuous Integration vs Delivery vs Deployment</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.guru99.com/ci-cd-pipeline.html'>CI/CD Pipeline: Learn with Example</BadgeLink>
```
